src/Nethermind/Nethermind.Merge.AuRa/AuRaMergePlugin.cs

```diff
@@ -19,19 +36,18 @@ namespace Nethermind.Merge.AuRa
     public class AuRaMergePlugin : MergePlugin, IInitializationPlugin
     {
         private AuRaNethermindApi? _auraApi;
-        private IAuRaMergeConfig? _auraMergeConfig;
 
-        public override bool MergeEnabled => _auraMergeConfig!.Enabled;
+        public override bool MergeEnabled => ShouldBeEnabled(_api);
 
         public override async Task Init(INethermindApi nethermindApi)
         {
-            _auraMergeConfig = nethermindApi.Config<IAuRaMergeConfig>();
-            if (_auraMergeConfig.Enabled)
+            _api = nethermindApi;
+            _mergeConfig = nethermindApi.Config<IMergeConfig>();
+            if (MergeEnabled)
             {
                 await base.Init(nethermindApi);
                 _auraApi = (AuRaNethermindApi)nethermindApi;
                 _auraApi.PoSSwitcher = _poSSwitcher;
-                _mergeConfig.Enabled = false; // set MergePlugin as disabled
```

Comment:
we could leave this but only falsify `_mergeConfig.Enabled` if `IsPreMergeConsensusAuRa`, it would be better as AuRa aware code would be moved to `AuRaMergePlugin` instead of being in the `MergePlugin`